build.sh

```diff
@@ -37,14 +39,21 @@ exit_error() {
 usage() {
 	cat <<-EOF
 	Usage: $(basename "${BASH_SOURCE[0]}") [-hu] [-k /path/to/authorized_keys]
-
+	
 	-h Show help
+	-r Release (valid values are 13.2 or 14.0)
 	-u Build with upload to remote host
 	-k /path/to/authorized_keys (can safely ignore, another opportunity to copy
 	   in SSH keys on image boot!)
 	EOF
 }
 
+# Check if no arguments were passed
+if [ $# -eq 0 ]; then
+    usage
+    exit 0
+fi
+
```

Comment:
This check  should go away (as we don't take arguments, just options, it should be checked below)